### ISSUE TITLE
Parsing error for rotate_right and rotate_left operations

### ISSUE THREAD

**Issue by rainoftime** [SRI-CSL/yices2#190]
Hi, for the following formula,
~~~~
(set-logic QF_UFBV)
(assert (= (_ bv53 6) (_ bv53 6) ((_ rotate_right 10) (_ bv53 6)) (_ bv53 6)))
~~~~

yices2 issues a warning 
~~~~
(error "at line 2, column 38: in rotate_right: invalid index in rotate")
~~~~
z3 and cvc4 can solve the formula correctly.

Is it because the formula is out of the SMT-Lib2 standard? Thank you!

**Comment by BrunoDutertre** [comment]
Why would anybody want to rotate a 6-bit vector by 10?

**Comment by BrunoDutertre** [comment]
Fixed anyway by 0611907df5dcfd1054e31e3e4c802dfd88736eaf.
